pkg/storaged/details.js

```diff
@@ -1728,8 +1728,11 @@
 
             if (html) {
                 $('#detail button.tooltip-ct').tooltip('destroy');
-                $('#detail').amend(html);
-                $('#detail button.tooltip-ct').tooltip();
+                $('#detail').html(html);
+                // if we don't use setTimeout here, sometimes not all tooltips will be updated
```

Comment:
So this is a HACK? Why is 'destroy' necessary above? What happens if you remove that.